stanford-earth/SE3_BLT
# EARTH-000: Remove settings.php and patch on each install/update with composer

# READY FOR REVIEW

# Summary
- Fixes bug where patch was being applied multiple times to settings.php and causing duplicate code.
- Uncomments the settings.local.php include code snippet in the new settings.php with the patch
- All local settings including DB connection will now have to be in the settings.local.php file and not in settings.php as it gets tossed on each composer install/update command.
- Deletes and re-creates the settings.php file each time composer install/update is ran.

# Needed By (Date)
- Sometime

# Steps to Test

1. Check out this branch
2. Run composer install or update
3. Validate settings.php file has been created and patched appropriately 
4. Create or add to settings.local.php your db connection settings
5. Validate site still works 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
